<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button00">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button01">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button02">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button03">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button04">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button05">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button06">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button07">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button14">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button17">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button15">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button16">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button13">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button32">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button31">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button36">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button24">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button20">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button30">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button22">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button27">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button34">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button25">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button33">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button21">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button37">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button26">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button35">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>410</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button23">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button52">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button51">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button56">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button44">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button40">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button50">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button42">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button47">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button54">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button45">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button53">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button41">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button57">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button46">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button55">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>550</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button43">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button72">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button71">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button76">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button64">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button60">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button70">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button62">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button67">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button74">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button65">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button73">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button61">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button77">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button66">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button75">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>690</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="button63">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>620</y>
      <width>68</width>
      <height>68</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
